okhttp/src/main/kotlin/okhttp3/internal/platform/Platform.kt

```diff
@@ -184,6 +184,8 @@ open class Platform {
     fun alpnProtocolNames(protocols: List<Protocol>) =
         protocols.filter { it != Protocol.HTTP_1_0 }.map { it.toString() }
 
+    val isAndroid = "Dalvik".equals(System.getProperty("java.vm.name"))
```

Comment:
Any background on how you came to this one? I don’t love it but I can’t think of anything better

Reply:
`==` instead of `.equals()` ?

Reply:
Took same approach as Retrofit fix. Cc @JakeWharton

Reply:
I had a change in OkHttp to implement this fix as well. Was waiting on the Retrofit PR review.

Reply:
Merged!

Reply:
Thanks. Is this going to make it into 3.14.x? Retrofit still depends on that version 😶 .

Reply:
Yep!

Reply:
🙏

Reply:
No offense intended, didn't see a PR in flight so thought it was easy enough to copy the same approach you used with Retrofit.